Tools/ci-run.sh

```diff
@@ -101,7 +101,7 @@ python runtests.py \
   --backends=$BACKEND \
    $LIMITED_API \
    $EXCLUDE \
-   $(if [ "$COVERAGE" == "1" ]; then echo " --coverage"; fi) \
+   $(if [ "$COVERAGE" == "1" ]; then echo " --coverage --coverage-html"; fi) \
```

Comment:
No need to actually run the test modules. For coverage, generating them (i.e. running Cython on them) is enough.
```suggestion
   $(if [ "$COVERAGE" == "1" ]; then echo " --coverage --coverage-html --cython-only"; fi) \
```